### PR TITLE
fix: Long package names are clipped

### LINKED ISSUE #1324: Long package names are clipped between columns on the sequence diagram view
### Issue

Reported by a community user working in Java:

> when I view the appmap - sequence diagram tab - the package names of my events are being cut off between the columns.

Image

### Details

* [This AppMap](https://github.com/getappmap/appmap-js/files/12530085/long-package.appmap.json.zip) demonstrates the issue.
* Please truncate the return value to remove unnecessary package and class names (e.g. `java.lang.Boolean` can be `Boolean`).
* Start looking in `packages/components/src/components/sequence/ReturnLabel.vue` for the relevant code.
* Please write a test in `packages/components/tests/e2e/specs` for this change.

### PR DESCRIPTION
**Pull request by zermelo-wisen**
Fixes #1324

Before
Pasted Graphic 2

After
Pasted Graphic 1

### CODE REVIEW

**Comment by ahtrotta** [comment]
@zermelo-wisen this looks good, but could you rebase it before I finish my review? Thanks!

**Comment by zermelo-wisen** [comment]
@ahtrotta, thank you. Is it ok now?

File: packages/components/src/stories/VsCodeExtension.stories.js [hunk 1]
```diff
@@ -28,6 +30,7 @@ const scenarioData = {
   longPackage,
   mapWithDiff: diffAppMap,
   longPackage,
+  mapWithDiff: diffAppMap,
```
Comment (kgilpin) [inline packages/components/src/stories/VsCodeExtension.stories.js:33]: `mapWithDiff` is in the map twice now.

**Comment by kgilpin** [comment]
@zermelo-wisen this PR has 28 commits, most of them are not yours. You need to fetch `getappmap:main` and then `git rebase` your branch to the remote, then re-push.

**Comment by kgilpin** [comment]
Looks good now - thanks

**Review by ahtrotta — APPROVED** [review]
Thanks!

**Pull request merged**